internal/services/eventhub/eventhub_resource_test.go

```diff
@@ -379,7 +379,7 @@ resource "azurerm_eventhub_namespace" "test" {
 
 resource "azurerm_eventhub" "test" {
   name                = "acctesteventhub-%d"
-  namespace_name      = azurerm_eventhub_namespace.test.name
+  namespace_id        = azurerm_eventhub_namespace.test.id
```

Comment:
I'm assuming all the test configs have been updated to use `namespace_id` instead of `namespace_name` now, provided the migration path for users switching from `namespace_name` -> `namespace_id` works this is fine but it might be worth keeping one test config with `namespace_name` to ensure it's still functional for the time being e.g.

```
func (EventHubResource) basic(data acceptance.TestData, partitionCount int) string {
     if !features.FivePointOhBeta() {
        // return config using namespace_name here
}

// return config using namespace_id here
}
```